index.js

```diff
@@ -1,10 +1,12 @@
 'use strict';
 
-const basename = require('path').basename;
-
 const UNNAMED = /import ['"]([^'"]+)['"];?/gi;
 const NAMED = /import (\{?)([\s\S]*?)\}? from ['"]([^'"]+)['"];?/gi;
 
+function basename(path) {
+	return path.split('/').pop();
```

Comment:
I don't suppose we need to support `\\` separators? I've never developed on Windows, so I don't know if Node-Windows users ever write import statements that way.

Reply:
As I understand they are URLs, so you're not allowed to use `\` there.